opentelemetry-sdk/src/logs/log_processor.rs

```diff
@@ -278,7 +282,8 @@ impl<R: RuntimeChannel> BatchLogProcessor<R> {
                             &timeout_runtime,
                             logs.split_off(0),
                         )
-                        .await;
+                        .await
+                        .and(exporter.as_mut().force_flush());
```

Comment:
Do we need this? `export_with_timeout` already calls `exporter.export` which is what is expected from `force_flush`.